crates/router/src/core/payments/transformers.rs

```diff
@@ -337,6 +339,15 @@ where
                         }));
                 };
 
+                // next action check for third party sdk session (for ex: Apple pay through trustpay has third party sdk session response)
+                if third_party_sdk_session_next_action(&payment_attempt, operation) {
+                    next_action_response = Some(
+                        api_models::payments::NextActionData::ThirdPartySdkSessionToken {
+                            session_token: session_tokens[0].clone(),
```

Comment:
`session_tokens[0]` could panic,
I'd prefer to use `session_tokens.first()` here...

Reply:
we should be updating our clippy rule with these
(don't add the clippy rule in this PR it might require of you to fix a lot more things)
```
#![forbid(clippy::indexing_slicing)]
```